project-generated-code-backend/Backend/Model/Schedule/Appointment.cs

```diff
@@ -30,14 +30,30 @@ public class Appointment : Entity
         public bool Active { get; set; }
         public string DateOfCanceling { get; set; }
 
+        public bool IsSurveyDone { get; set; }
+        
         public Appointment(Room room, Physician physician, Patient patient, TimeInterval timeInterval,
-            ProcedureType procedureType) : base()
+            ProcedureType procedureType, Boolean active, DateTime date) : base()
```

Comment:
Please create a new constructor that contains active and date, then you won't need to comment the backend code 😄

Reply:
We need additional explanation, we do not understand what the problem is.